### PR TITLE
Check image dimensions, and, if the image aspect is closer to the the…

### PR DESCRIPTION
**Pull request by extorn**
Check image dimensions, and, if the image aspect is closer to the the screen's aspect when rotated 90 degrees, do that. Configurable with a setter method - setRotateImageToFitScreen

### CODE REVIEW

File: touchview/src/main/java/com/ortiz/touchview/TouchImageView.java [hunk 1]
```diff
@@ -1,1463 +1,1511 @@
-package com.ortiz.touchview;
-
-import android.annotation.TargetApi;
-import android.content.Context;
```
Comment (hannesa2) [inline touchview/src/main/java/com/ortiz/touchview/TouchImageView.java]: What happens ? You changed **every** line, a valid diff is impossible !
Maybe wrong line separator ?

Please double check it 
image

**Comment by extorn** [comment]
Hi,Sorry to have caused such trouble for you. I diff'd the file before commit and there were certainly not many lines changed. I guess my intellij has some pre commit rules I've forgotten about.I'll have a look at it when next on my computer. GarethOn 9 Sep 2019 8:31 pm, hannesa2 <notifications@github.com> wrote:@hannesa2 requested changes on this pull request.



In touchview/src/main/java/com/ortiz/touchview/TouchImageView.java:
> @@ -1,1463 +1,1511 @@
-package com.ortiz.touchview;
-
-import android.annotation.TargetApi;
-import android.content.Context;

What happens ? You changed every line, a valid diff is impossible !
Maybe wrong line separator ?
Please double check it

**Comment by extorn** [comment]
I've made the requested change allowing diff without "ignore white-space selected". Could you please look carefully at the block starting line 666. I'm not sure whether this is something I altered or you've since changed, but, it changes the logic in that the previous values don't get saved if it just returns. Happy to remove this change and re-commit if it isn't correct.

If there are features of this class that are affected that my changes are not complete for - e.g. getZoomedRect has not been tested, please let me know.

**Comment by hannesa2** [comment]
Thank you

**Pull request merged**